testflinger_agent/tests/test_client.py

```diff
@@ -14,11 +14,14 @@
 
 import pytest
 import uuid
+import influxdb
 
 import requests_mock as rmock
 
 from testflinger_agent.client import TestflingerClient as _TestflingerClient
 
+del influxdb
```

Comment:
why?

Reply:
Wasn't clear on what the unit test was complaining about last night. This can be discarded. :)